settings.gradle.kts

```diff
@@ -11,3 +12,10 @@ dependencyResolutionManagement {
         mavenCentral()
     }
 }
+
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
```

Comment:
This is needed for the Native Build Tools Gradle plugin: 
https://graalvm.github.io/native-build-tools/latest/gradle-plugin.html#_adding_the_plugin